### PR TITLE
Creating the UI of  calculator

### PR DESCRIPTION
**Pull request by Kidd254**
**In this pull request I: **

- [ ] Delete dall extra files from react-creat-app command.

   

- [ ]  Created components folder inside src.

    

- [ ] Create Calculator.js.

    

- [ ] Created multiple child and exported Calculator.

    

- [ ] Imported Calculator from App.js.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+      }}
+    >
+      <div
+        style={{
+          display: 'flex',
+          height: 50,
+          flex: 1,
+        }}
+      >
+        <Display numberParent={0} />
```
Comment (divinecharlotte) [inline src/components/Calculator.js:24]: - [ ] kindly consider sticking to the given design by placing this `numberParent` to the right of the grey container to make sure that your design implements all the design guideline
- your design 👇 
Screen Shot 2023-07-04 at 22 41 45
- expected design 👇 
Screen Shot 2023-07-04 at 22 42 45

**Review by divinecharlotte — CHANGES_REQUESTED** [review]
Hi @Kidd254 , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## To Highlight 💯 

- Linters are passing 👍 
- Descriptive pr👍 

 




 

## Required Changes ♻️

- [ ] kindly consider adding a working link to your LinkedIn account under the author's section to make your readme more interactive and professional

- [ ] Kindly consider making sure that you removed  All the boilerplate that got created by the CRA (texts, images, and styles) hint 👉 remove `logo.svg`file and `App.css` 
to meet the below requirement 👇 
> Delete all the boilerplate from Create React App (text, images, styles).
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.css [hunk 2]
```diff
@@ -1 +1,3 @@
-
+* {
+  margin-left: 0;
+}
```
Comment (Munsa1) [inline src/App.css:3]: - [ ] As requested by the previous reviewer, kindly remove `App.js` as it is boilerplate code.

File: src/components/Calculator.js [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+
+Display.defaultProps = {
+  numberParent: 0,
+};
+
+Display.propTypes = {
+  numberParent: PropTypes.number,
+};
+
+export default Calculator;
```
Comment (Munsa1) [inline src/components/Calculator.js:190]: - [ ] (optional). I would advise you to separate your CSS from your Components so as to follow best practices. It makes it easier to manage your codebase as it grows and it makes it cleaner and more readable to other users.

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @Kidd254, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
Fixed Design As Requested :heavy_check_mark: 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
### Due to insufficient files changed 
- [ ] You could remove some more boilerplate code such as `reportWebVitals`,  and `App.test.js`

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mukhammadrizooff — APPROVED** [review]
Hi @Kidd254, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

 

## Highlights
- No linter errors ✔️ 
- Correct GitFlow ✔️ 
- Good implementations ✔️
- Descriptive `README` file ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**